### ISSUE TITLE
multivariate prediction capability

### ISSUE THREAD

**Issue by Cuiyingzhe** [neuraloperator/neuraloperator#124]
Hello, I'm new to FNO and I saw nice results in univariate field prediction (e.g., the vorticity $\omega$ in 2D NS eq.).

I'm wondering if FNO is able to model multivariate fields, say the $(x,y,z)$ in the Lorenz 63 system[1]:
$\displaystyle\frac{d x}{d t}=\sigma(y-x)$
$\displaystyle\frac{d y}{d t}=x(r-z)-y$
$\displaystyle\frac{d z}{d t}=x y-b z$
where $\sigma, r, b$ are real constants.

In this case, similar to the case in fourier_3d.py, I want to utilize FNO to map this system from its preceding states $(x,y,z)\mathrm{[...,:T_{in}]}$ to the subsequent $(x,y,z)\mathrm{[...,T_{in}:T+T_{in}]}$.

Will it work if I simply concatenate $(x,y,z)$ along the time axis, 
 like `A=torch.cat((x,y,z), dim=-1)`, and then feed $\mathrm{A}$ into FNO?

Looking for comments and advice. Thanks in advance for any help.
 
[1] Lorenz, Edward N., 1963: Deterministic Nonperiodic Flow. Journal of the Atmospheric Sciences, 20, 130-141.

**Comment by zongyi-li** [comment]
Lorenz 63 is an ODE system, where the state (x,y,z) is in R^3. On the other hand, FNO is designed for PDE where the state is a field U[x,y,z] which is usually discretized into a tensor, say, R^{64x64x64}.

The closest form is to view the time dimension as a spatial dimension U[t], and use the fourier_1d.py code. But since the lorenz equation is chaotic, we won't expect T too be long.

**Comment by Cuiyingzhe** [comment]
Thanks for your informative comment. I see now FNO is designed for PDE.

I have another question. What if a PDE contains multiple fields of interest, like U[x,y,t], V[x,y,t], W[x,y,t] and each of them is discretized into a tensor, say, R^{64x64x20}. 

Since `fourier_3d.py` is to model univariate field (the vorticity \omega), could `fourier_3d.py` to be extended to `fourier_4d` to allow multiple fields as its input and output? Like adding another dimension in `SpectralConv3d` class. Or maybe just model U, V, W separately.

Any comment would be appreciated.

**Comment by Mencu** [comment]
Hi. 
I was wondering how you modeled the Lorenz equations with FNOs in the end? I would also be interested in solving ODEs with FNOs. Did treating the time dimension as a spatial dimension give any results?
